packages/sodium/sodium.0.2.0/opam

```diff
@@ -11,6 +11,7 @@ depends: [
   "base-bytes"
   "ocamlfind"
   "ctypes" {>= "0.3"}
+  "ctypes-foreign"
```

Comment:
I think so.  The split packages for pinning, i.e. being able to say

```
opam pin add ctypes https://github.com/ocamllabs/ocaml-ctypes.git
opam pin add ctypes-foreign https://github.com/ocamllabs/ocaml-ctypes.git
```

requires OPAM 1.2.1, but I don't think that anything here requires recent a particularly OPAM.